### PR TITLE
Refactor/pm20 266/modal header

### PR DESCRIPTION
**Pull request by mati-1**
[PM20-266](https://playmakerpro.atlassian.net/browse/PM20-266)

### CODE REVIEW

File: components/common/Modal/header.tsx [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 
 interface IProps extends Pick<IModalProps, 'title' | 'closeModal' | 'caption'> {
   showBar: boolean
   headerContent?: React.ReactNode
-  headerCloseIcon?: React.ReactNode
+  withClosingArrow?: boolean
+  withClosingCross?: boolean
+  searchValue?: string
+  setSearchValue?: Dispatch<SetStateAction<string>>
 }
```
Comment (dejwi) [inline components/common/Modal/header.tsx:17]: if you've added these props in IModalProps already, just add them to Pick

File: components/common/Modal/modal.tsx [hunk 2]
```diff
@@ -65,9 +69,12 @@ const Modal = ({
...
                 <ModalHeader
                   headerContent={headerContent}
-                  {...props}
                   showBar={showHeaderBar}
-                  headerCloseIcon={headerCloseIcon}
+                  withClosingArrow={withClosingArrow}
+                  searchValue={searchValue}
+                  setSearchValue={setSearchValue}
+                  withClosingCross={withClosingCross}
+                  {...props}
```
Comment (dejwi) [inline components/common/Modal/modal.tsx:77]: you can remove your changes from this file, if you won't separate these props they'll be passed with `{...props}`

**Review by dejwi — CHANGES_REQUESTED** [review]
you have to update styles as well bro, if you check the clubs modal search at least on mobile it should have smaller py and search&arrow are too far apart, on desktop arrow should be bigger

check things yourself too maybe there's something else

File: components/common/Modal/header.tsx (hunk 1 above)
Comment (mati-1) [reply components/common/Modal/header.tsx:17]: fr

File: components/common/Modal/modal.tsx (hunk 2 above)
Comment (mati-1) [reply components/common/Modal/modal.tsx:77]: 🤝

**Pull request closed without merging**